src/main.ts

```diff
@@ -201,7 +203,8 @@ export async function run(): Promise<void> {
       if (timeout) cmd.push('--timeout', timeout);
 
       // Compile the labels
-      const compiledLabels = Object.assign({}, defaultLabels(), labels);
+      const defLabels = skipDefaultLabels ? {} : defaultLabels();
+      const compiledLabels = Object.assign({}, defLabels, labels);
       cmd.push('--update-labels', kvToString(compiledLabels));
```

Comment:
does this command clear out all labels anyway when they are all empty? 
If there are no labels, I would have imagined this cmd shouldn't be invoked at all